RealmSwift-swift2.2/LinkingObjects.swift

```diff
@@ -111,7 +111,7 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
         super.init(fromClassName: className, property: propertyName)
     }
 
-    /// Returns a description of the objects contained within the linking objects.
+    /// Returns a description of the objects represented by the linking objects.
```

Comment:
As above; the phrase "the objects represented by the linking objects" just doesn't really work.